### PR TITLE
ci: upgrade action & use nextest instead of test for ut

### PR DESCRIPTION
**Pull request by ben1009**
Signed-off-by: ben1009 <liuhe1009@gmail.com>

### CODE REVIEW

**Comment by ben1009** [comment]
no code change, why, flaky?

**Comment by ben1009** [comment]
https://github.com/tikv/client-rust/issues/423

File: Makefile [hunk 1]
```diff
@@ -20,7 +20,7 @@ check: generate
 	cargo clippy --all-targets --features "${ALL_FEATURES}" -- -D clippy::all
 
 unit-test: generate
-	cargo test --all --no-default-features
+	cargo nextest run --all --no-default-features
```
Comment (pingyu) [inline Makefile:23]: Suggest to also add command for installing `nextest` in `Makefile`. Otherwise running unit tests locally would fail.

Rest LGTM~

Comment (ben1009) [reply Makefile:23]: add the installation guid for `nextest` in README, just for simplicity fellow what `tiup` did, which also a prerequisite for run IT.

**Review by pingyu — APPROVED** [review]
LGTM~

**Pull request merged**